app/controllers/exports_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ExportsController < ApplicationController
```

Comment:
In trying this out, I see that it is causing some very bizarre errors... as best I can tell, the spreadsheet gem or one of its dependencies seems to be causing it...? Very odd...............